nilearn/region.py

```diff
@@ -9,6 +9,7 @@
 
 import numpy as np
 from scipy import linalg, ndimage
+from six.moves import xrange
```

Comment:
`xrange` is a builtin in Python 2.x, but unavailable in Python 3.x.  Converting to `range` seemed suboptimal, since some of the ranges being iterated over are quite large.  So I added the imports so that the `xrange` function would be available in both.

Do you prefer simply converting all `xrange` to `range` then?